package.json

```diff
@@ -775,11 +775,22 @@
           "type": "string",
           "enum": [
             "FromPreference",
+            "Minimal",
             "Normal",
-            "Minimal"
+            "Diagnostic"
```

Comment:
What happens in the case where the default is used but Pester 5 isn't loaded (i.e. Pester 4.x is used)?

Reply:
https://github.com/PowerShell/vscode-powershell/blob/0a1dab2674fffbe6ee0e4f0384255f5681e34bae/InvokePesterStub.ps1#L83 It uses the default All.